### PR TITLE
Update Touch() contructor syntax

### PR DESCRIPTION
**Pull request by AnilSeervi**
- use `var`

### CODE REVIEW

File: files/en-us/web/api/touch/touch/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre> <em>touch</em> = new Touch(<em>touchInit</em>);</pre>
+<pre>var <em>touch</em> = new Touch(<em>touchInit</em>);</pre>
```
Comment (sideshowbarker) [inline files/en-us/web/api/touch/touch/index.html:18]: Actually, I think the convention we’ve adopted is to not try to show runnable code in Syntax sections — and so, show no assignments at all (and thus also no arbitrary variable names) but instead just show the syntax.

Therefore:

```suggestion
<pre>new Touch(<em>touchInit</em>)</pre>
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/api/touch/touch/index.html (hunk 1 above)
Comment (AnilSeervi) [reply files/en-us/web/api/touch/touch/index.html]: I see. I just followed the syntax shown on other pages.

Comment (sideshowbarker) [reply files/en-us/web/api/touch/touch/index.html]: > I see. I just followed the syntax shown on other pages.

Yeah, all the other pages need to be updated eventually too.

But the JavaScript docs have all already been updated — so if you want to see examples of what the convention is, take a look at the Syntax sections in the JavaScript articles.

Comment (AnilSeervi) [reply files/en-us/web/api/touch/touch/index.html]: Sure. Will have a look.

**Pull request merged**